clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -460,6 +460,49 @@ class radix_sort {
         break;
     }
 
+#pragma unroll
+    for (int i = 0; i < VALUES_PER_THREAD; ++i) {
+      unsigned_keys[i] = detail::traits<T>::twiddle_out(unsigned_keys[i]);
+    }
+  }
+
+  template <typename Item>
+  __dpct_inline__ void
+  sortBlockedToStriped(const Item &item, T (&keys)[VALUES_PER_THREAD],
+                       int begin_bit = 0, int end_bit = 8 * sizeof(T)) {
+
+    uint32_t(&unsigned_keys)[VALUES_PER_THREAD] =
+        reinterpret_cast<uint32_t(&)[VALUES_PER_THREAD]>(keys);
```

Comment:
Why should we convert to uint32_t instead of using T directly?

Reply:
How was this resolved?  
(In general, I'd recommend letting the reviewer resolve threads when they are satisfied, rather than resolving them yourself.)